index.js

```diff
@@ -70,12 +70,21 @@ function clicked(event: MouseEvent) {
   // Ignore clicks from nested details.
   if (target.closest('details') !== details) return
 
-  const item = target.closest('[role="menuitem"]')
+  const item = target.closest('[role^="menuitem"]')
   if (item) commit(item, details)
 }
 
+function updateChecked(selected: Element, details: Element) {
+  if (selected.getAttribute('role') !== 'menuitemradio') return
```

Comment:
How come we don't set check for `menuitemcheckbox`?

Reply:
411590f833af6dfb79773a664977909772cb3297